From a56ac2803a227a1983f58d501ea61127dee18f56 Mon Sep 17 00:00:00 2001
From: Tom Stellard <tstellar@redhat.com>
Date: Wed, 19 Apr 2023 16:05:58 -0700
Subject: [PATCH] [llvm-exegesis][AArch64] Only run tests on aarch64 hosts

The add-return.s test is failing on s390x.

See also e30ce634f75c01cc8784cb0c4972c42987178c1d.
---
 llvm/test/tools/llvm-exegesis/AArch64/lit.local.cfg | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/llvm/test/tools/llvm-exegesis/AArch64/lit.local.cfg b/llvm/test/tools/llvm-exegesis/AArch64/lit.local.cfg
index 2c80c84ef2cc..c1c887bfcc47 100644
--- a/llvm/test/tools/llvm-exegesis/AArch64/lit.local.cfg
+++ b/llvm/test/tools/llvm-exegesis/AArch64/lit.local.cfg
@@ -1,3 +1,6 @@
 if not ('AArch64' in config.root.targets):
     # We need support for AArch64.
     config.unsupported = True
+elif not ('aarch64' in config.root.host_triple):
+    # We need to be running on an PPC host.
+    config.unsupported = True
-- 
2.31.1

